.github/workflows/test-and-deploy.yml

```diff
@@ -28,12 +28,27 @@ jobs:
 
       - name: Build & Test
         run: make test
+      
+      - name: Run Cluster Test
```

Comment:
If sonar coverage also runs cluster tests, is this step needed?

Reply:
addressed